src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/config/EmailConfiguration.kt

```diff
@@ -54,7 +55,7 @@ abstract class Email(
   comments: String? = "",
 ) {
   private val common = mapOf(
-    "date" to date.toString(),
+    "date" to date.toMediumFormatStyle(),
```

Comment:
This is the style used in the templates.  Note we have to do the formatting here.